packages/components/addon/components/hds/pagination-bar/page-size.hbs

```diff
@@ -0,0 +1,10 @@
+<div class="hds-pagination-page-size hds-typography-body-200 hds-font-weight-medium" ...attributes>
+  <label for={{this.pageSizeId}}>Items per page</label>
```

Comment:
since the typographic styles should only apply to the label, better to move them where their actual application context is:
```suggestion
<div class="hds-pagination-page-size" ...attributes>
  <label for={{this.pageSizeId}} class="hds-pagination-page-size hds-typography-body-200 hds-font-weight-medium">Items per page</label>
```

Reply:
Thanks, I'll look at these. I haven't started implementing responsive layouts yet.